.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: Close Stale Issues
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+    inputs:
+      readonly:
+        description: "readonly: Specify true or 1 to prevent changes from being commited to GitHub"
+        default: false
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      stale-days: 60
+      close-days: 7
+      stale-label: stale
+    steps:
+    - uses: actions/stale@v4.1.1
+      with:
+        stale-pr-label: $stale-label
```

Comment:
Do we want to mark PR's stable as well? If not, I doubt we need to define this variable.

Reply:
You're right, It should be `stale-issue-label`. fixing

Reply:
fixed